nautobot/dcim/models/power.py

```diff
@@ -137,31 +174,119 @@ class PowerFeed(PrimaryModel, PathEndpoint, CableTermination):
         "voltage",
         "amperage",
         "max_utilization",
+        "breaker_position",
+        "breaker_pole_count",
         "available_power",
     ]
 
     class Meta:
         ordering = ["power_panel", "name"]
         unique_together = ["power_panel", "name"]
```

Comment:
Maybe add `power_panel, breaker_position` as a unique constraint as well? (I know that won't cover all of the cases you're handling in `clean()` below but might be a useful start?)